x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1018,11 +1019,56 @@ describe('when on the endpoint list page', () => {
         expect(activityLogCallout).not.toBeNull();
       });
 
+      it('should not display scroll trigger when showing callout message', async () => {
+        const userChangedUrlChecker = middlewareSpy.waitForAction('userChangedUrl');
+        reactTestingLibrary.act(() => {
+          history.push(
+            getEndpointDetailsPath({
+              page_index: '0',
+              page_size: '10',
+              name: 'endpointActivityLog',
+              selected_endpoint: '1',
+            })
+          );
+        });
+        const changedUrlAction = await userChangedUrlChecker;
+        expect(changedUrlAction.payload.search).toEqual(
+          '?page_index=0&page_size=10&selected_endpoint=1&show=activity_log'
+        );
+        await middlewareSpy.waitForAction('endpointDetailsActivityLogChanged');
+        reactTestingLibrary.act(() => {
+          dispatchEndpointDetailsActivityLogChanged('success', {
+            page: 1,
+            pageSize: 50,
+            startDate: 'now-1d',
+            endDate: 'now',
+            data: [],
+          });
+        });
+
+        const activityLogCallout = await renderResult.findByTestId('activityLogNoDataCallout');
+        expect(activityLogCallout).not.toBeNull();
+        // scroll to the bottom by pressing down arrow key
+        // and keep it pressed
+        userEvent.keyboard('ArrowDown>');
+        // wait for key up
+        await waitFor(() => {
```

Comment:
```suggestion
        await waitFor();
        userEvent.keyboard('/ArrowDown');
        expect(await renderResult.queryByTestId('activityLogLoadMoreTrigger')).toBeNull();
```

Check if it works but according to the documentation, the implicit timeout for waitFor is 1000ms and should be enough time to wait before release the key right?

Reply:
done c8a9c494e9383e169188c1e504ad8f6dcd418cd3